variants/esp320/pins_arduino.h

```diff
@@ -9,6 +9,12 @@
 #define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
 #define digitalPinHasPWM(p)         (p < 34)
 
+static const uint8_t LED_BUILTIN = 5;
+static const uint8_t BUILTIN_LED = 5;
```

Comment:
Yes, you are right, I will do it. I tried to do it same way as is in ESP8266 core.